### PR TITLE
002 crud operations

### PR DESCRIPTION
**Pull request by bleuflame47**
Trying to get up to date with master

### CODE REVIEW

**Review by kuddlefishbwaa — CHANGES_REQUESTED** [review]
Looking more into the file, the CSV file only hast 6000~ entries so we don't need to change the db table to something different

File: DataProject/DatabaseWarehouse.py [hunk 1]
```diff
@@ -19,63 +19,18 @@ class databaseWarehouse():
     - Exit (Ask if you want to save if any changes were made)
     """
 
-    def __init__(self, master, **kwargs):
```
Comment (kuddlefishbwaa) [inline DataProject/DatabaseWarehouse.py:22]: I don't think that we should delete all of this work. It's the whole base that brings up the DB in a view window. /the replacement would only have a button of "What would you like to do?"

File: DataProject/Operations.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ def formatTable(self):
         """Fixer method that is to format the table, use only if the table is not in the DB. """
         
         csv_file_name = 'inventory_team6.csv'
+        #  I'm getting an FileNotFoundError on the line below.  Any suggestions as to how to fix this?
```
Comment (kuddlefishbwaa) [inline DataProject/Operations.py:40]: Was this fixed? if so then we can delete that.

**Comment by Stowne1** [comment]
Helped test and review code.

**Pull request closed without merging**